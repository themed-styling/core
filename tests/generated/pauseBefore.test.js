import 'jest-styled-components'

import { pauseBefore } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(pauseBefore, 'pauseBefore')
