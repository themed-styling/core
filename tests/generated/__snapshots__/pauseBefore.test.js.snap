// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pauseBefore renders 0 1`] = `
Array [
  .c0 {
  -webkit-pause-before: 0;
  -moz-pause-before: 0;
  -ms-pause-before: 0;
  pause-before: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: 0;
  -moz-pause-before: 0;
  -ms-pause-before: 0;
  pause-before: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: 0;
  -moz-pause-before: 0;
  -ms-pause-before: 0;
  pause-before: 0;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: 0;
  -moz-pause-before: 0;
  -ms-pause-before: 0;
  pause-before: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`pauseBefore renders function 1`] = `
Array [
  .c0 {
  -webkit-pause-before: hasnot;
  -moz-pause-before: hasnot;
  -ms-pause-before: hasnot;
  pause-before: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: has;
  -moz-pause-before: has;
  -ms-pause-before: has;
  pause-before: has;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: hasnot;
  -moz-pause-before: hasnot;
  -ms-pause-before: hasnot;
  pause-before: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: has;
  -moz-pause-before: has;
  -ms-pause-before: has;
  pause-before: has;
}

<div
    className="c0"
  />,
]
`;

exports[`pauseBefore renders invalid theme path 1`] = `
Array [
  .c0 {
  -webkit-pause-before: invalid.path.1;
  -moz-pause-before: invalid.path.1;
  -ms-pause-before: invalid.path.1;
  pause-before: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: invalid.path.1;
  -moz-pause-before: invalid.path.1;
  -ms-pause-before: invalid.path.1;
  pause-before: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: invalid.path.1;
  -moz-pause-before: invalid.path.1;
  -ms-pause-before: invalid.path.1;
  pause-before: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: invalid.path.1;
  -moz-pause-before: invalid.path.1;
  -ms-pause-before: invalid.path.1;
  pause-before: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`pauseBefore renders negative number 1`] = `
Array [
  .c0 {
  -webkit-pause-before: -54881;
  -moz-pause-before: -54881;
  -ms-pause-before: -54881;
  pause-before: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: -54881;
  -moz-pause-before: -54881;
  -ms-pause-before: -54881;
  pause-before: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: -42365;
  -moz-pause-before: -42365;
  -ms-pause-before: -42365;
  pause-before: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: -42365;
  -moz-pause-before: -42365;
  -ms-pause-before: -42365;
  pause-before: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`pauseBefore renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: July;
    -moz-pause-before: July;
    -ms-pause-before: July;
    pause-before: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: August;
    -moz-pause-before: August;
    -ms-pause-before: August;
    pause-before: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: September;
    -moz-pause-before: September;
    -ms-pause-before: September;
    pause-before: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-pause-before: November;
    -moz-pause-before: November;
    -ms-pause-before: November;
    pause-before: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: July;
    -moz-pause-before: July;
    -ms-pause-before: July;
    pause-before: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: August;
    -moz-pause-before: August;
    -ms-pause-before: August;
    pause-before: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: September;
    -moz-pause-before: September;
    -ms-pause-before: September;
    pause-before: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-pause-before: November;
    -moz-pause-before: November;
    -ms-pause-before: November;
    pause-before: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: May;
    -moz-pause-before: May;
    -ms-pause-before: May;
    pause-before: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: June;
    -moz-pause-before: June;
    -ms-pause-before: June;
    pause-before: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: April;
    -moz-pause-before: April;
    -ms-pause-before: April;
    pause-before: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-pause-before: November;
    -moz-pause-before: November;
    -ms-pause-before: November;
    pause-before: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: May;
    -moz-pause-before: May;
    -ms-pause-before: May;
    pause-before: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: June;
    -moz-pause-before: June;
    -ms-pause-before: June;
    pause-before: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    -webkit-pause-before: April;
    -moz-pause-before: April;
    -ms-pause-before: April;
    pause-before: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    -webkit-pause-before: November;
    -moz-pause-before: November;
    -ms-pause-before: November;
    pause-before: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`pauseBefore renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    -webkit-pause-before: July;
    -moz-pause-before: July;
    -ms-pause-before: July;
    pause-before: July;
  }
}

@media April {
  .c0 {
    -webkit-pause-before: August;
    -moz-pause-before: August;
    -ms-pause-before: August;
    pause-before: August;
  }
}

@media January {
  .c0 {
    -webkit-pause-before: September;
    -moz-pause-before: September;
    -ms-pause-before: September;
    pause-before: September;
  }
}

@media April {
  .c0 {
    -webkit-pause-before: November;
    -moz-pause-before: November;
    -ms-pause-before: November;
    pause-before: November;
  }
}

@media June {
  .c0 {
    -webkit-pause-before: August;
    -moz-pause-before: August;
    -ms-pause-before: August;
    pause-before: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    -webkit-pause-before: July;
    -moz-pause-before: July;
    -ms-pause-before: July;
    pause-before: July;
  }
}

@media April {
  .c0 {
    -webkit-pause-before: August;
    -moz-pause-before: August;
    -ms-pause-before: August;
    pause-before: August;
  }
}

@media January {
  .c0 {
    -webkit-pause-before: September;
    -moz-pause-before: September;
    -ms-pause-before: September;
    pause-before: September;
  }
}

@media April {
  .c0 {
    -webkit-pause-before: November;
    -moz-pause-before: November;
    -ms-pause-before: November;
    pause-before: November;
  }
}

@media June {
  .c0 {
    -webkit-pause-before: August;
    -moz-pause-before: August;
    -ms-pause-before: August;
    pause-before: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-pause-before: January;
    -moz-pause-before: January;
    -ms-pause-before: January;
    pause-before: January;
  }
}

@media April {
  .c0 {
    -webkit-pause-before: December;
    -moz-pause-before: December;
    -ms-pause-before: December;
    pause-before: December;
  }
}

@media January {
  .c0 {
    -webkit-pause-before: April;
    -moz-pause-before: April;
    -ms-pause-before: April;
    pause-before: April;
  }
}

@media April {
  .c0 {
    -webkit-pause-before: May;
    -moz-pause-before: May;
    -ms-pause-before: May;
    pause-before: May;
  }
}

@media June {
  .c0 {
    -webkit-pause-before: June;
    -moz-pause-before: June;
    -ms-pause-before: June;
    pause-before: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    -webkit-pause-before: January;
    -moz-pause-before: January;
    -ms-pause-before: January;
    pause-before: January;
  }
}

@media April {
  .c0 {
    -webkit-pause-before: December;
    -moz-pause-before: December;
    -ms-pause-before: December;
    pause-before: December;
  }
}

@media January {
  .c0 {
    -webkit-pause-before: April;
    -moz-pause-before: April;
    -ms-pause-before: April;
    pause-before: April;
  }
}

@media April {
  .c0 {
    -webkit-pause-before: May;
    -moz-pause-before: May;
    -ms-pause-before: May;
    pause-before: May;
  }
}

@media June {
  .c0 {
    -webkit-pause-before: June;
    -moz-pause-before: June;
    -ms-pause-before: June;
    pause-before: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`pauseBefore renders postitive number 1`] = `
Array [
  .c0 {
  -webkit-pause-before: 54881;
  -moz-pause-before: 54881;
  -ms-pause-before: 54881;
  pause-before: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: 54881;
  -moz-pause-before: 54881;
  -ms-pause-before: 54881;
  pause-before: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: 42365;
  -moz-pause-before: 42365;
  -ms-pause-before: 42365;
  pause-before: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: 42365;
  -moz-pause-before: 42365;
  -ms-pause-before: 42365;
  pause-before: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`pauseBefore renders string 1`] = `
Array [
  .c0 {
  -webkit-pause-before: July;
  -moz-pause-before: July;
  -ms-pause-before: July;
  pause-before: July;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: July;
  -moz-pause-before: July;
  -ms-pause-before: July;
  pause-before: July;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: June;
  -moz-pause-before: June;
  -ms-pause-before: June;
  pause-before: June;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: June;
  -moz-pause-before: June;
  -ms-pause-before: June;
  pause-before: June;
}

<div
    className="c0"
  />,
]
`;

exports[`pauseBefore renders valid theme path 1`] = `
Array [
  .c0 {
  -webkit-pause-before: DarkSlateBlue;
  -moz-pause-before: DarkSlateBlue;
  -ms-pause-before: DarkSlateBlue;
  pause-before: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: DarkSlateBlue;
  -moz-pause-before: DarkSlateBlue;
  -ms-pause-before: DarkSlateBlue;
  pause-before: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: DarkSlateBlue;
  -moz-pause-before: DarkSlateBlue;
  -ms-pause-before: DarkSlateBlue;
  pause-before: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  -webkit-pause-before: DarkSlateBlue;
  -moz-pause-before: DarkSlateBlue;
  -ms-pause-before: DarkSlateBlue;
  pause-before: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
